src/session.ts

```diff
@@ -0,0 +1,70 @@
+/* Made by the cracklR Group in '24!
+Session objects help keep track of an ongoing game between two players.
+It expires after 1 hour, when it expires, the game will end on it's own, no EXP is awarded to either player.
+*/
+
+type onEndListener = (sessionObj: session) => void;
+
+export default class session {
```

Comment:
The convention is that class names should be capitalized.

Reply:
hmm... didn't realize that was a convention. If we want to do that we can

Reply:
Yeah I'd say it helps, so you know you are dealing with a class/interface/type/etc., instead of a variable.